editoast/migrations/2024-01-17-140850_rollingstock-add-supported-signaling-systems/up.sql

```diff
@@ -0,0 +1,4 @@
+-- Your SQL goes here
+
+ALTER TABLE rolling_stock
+ADD supported_signaling_system jsonb NOT NULL DEFAULT ('["BAPR", "BAL", "TVM300", "TVM430"]');
```

Comment:
So by default, all current rolling stocks support TVM?

Reply:
We adjusted to only have TGVs supporting TVM for now, until we get better data

Reply:
the data is in